glomap/controllers/rotation_averager.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "glomap/estimators/global_rotation_averaging.h"
+
+namespace glomap {
+
+struct RotationAveragerOptions : public RotationEstimatorOptions {
+  bool use_stratified = true;
+};
+
+class RotationAverager {
```

Comment:
So, this only solves the gravity aligned rotation averaging problem and is not a general interface for rotation averaging? Why is this a separate, new class? Otherwise, a more specific name would be preferable?

Reply:
No, the interface is for both 3DoF RA and 1DoF RA. If gravity direction is not provided, then the original 3DoF will be applied

Reply:
Thanks for clarifying. It feels odd that this is a class. Why not just make it a function?